docs/docs/end-to-end-testing.md

```diff
@@ -49,10 +51,123 @@ If you want to run Cypress in Continuous Integration (CI) you have to use `cypre
   "scripts": {
     "develop": "gatsby develop",
     "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "test:e2e:ci": "start-server-and-test develop http://localhost:8000 cy:run"
+    "cy:run": "cypress run", // highlight-line
+    "test:e2e": "start-server-and-test develop http://localhost:8000 cy:open"
+    "test:e2e:ci": "start-server-and-test develop http://localhost:8000 cy:run" // highlight-line
   }
 }
 ```
 
 Please read the [Cypress' official documentation](https://docs.cypress.io/guides/guides/continuous-integration.html) on CI if you want to know how to setup Travis or GitLab with Cypress.
+
+## Writing tests
+
+The first test is an **accessibility** test with [cypress-axe](https://github.com/avanslaars/cypress-axe).
```

Comment:
```suggestion
A good use case for writing automated end-to-end tests is asserting **accessibility** with [cypress-axe](https://github.com/avanslaars/cypress-axe), a Cypress plugin that incorporates the [axe](https://deque.com/axe) accessibility testing API. While some [manual testing](https://www.smashingmagazine.com/2018/09/importance-manual-accessibility-testing/) is still required to ensure good web accessibility, automation can ease the burden on human testers.
```
I think this needs to be introduced a bit more generally. It also provides an opportunity to discuss the both the benefits and limits of accessibility automation.